README.md

```diff
@@ -48,3 +48,63 @@ Requests to cross-scheme subresources will no longer send SameSite=Lax or SameSi
 
 For example, if http://site.example embeds an image from https://site.example, SameSite=Lax and SameSite=Strict cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
+## Questions
+### How do Schemeful Same-Site and Scheme-Bound Cookies differ?
+
+"Schemeful Same-Site" and "[Scheme-Bound Cookies](https://github.com/mikewest/scheming-cookies)" are both trying to move cookies closer to an origin based security model; these two proposals complement one another and one is not a subset of the other.
+
+#### Schemeful Same-Site
+"Schemeful Same-Site", and the current version of same-site, is concerned with the browsing context in which the cookie will be sent or recieved. The browsing context can be thought of as the answer to the question "Is the request/response url the same site as the one I'm on?" More specifically "Is the request/response url 'same-site' or 'cross-site' with the current site I'm on?" 
+
+In the current world the browser context is determined by checking the registrable domain: http://site1.example is *not* same-site with http://site2.example, but http://site1.example and https://site1.example would be same-site.
```

Comment:
Browsing context has a specific meaning in the HTML spec. (It's a live rendered document.) I don't think it fits here. SameSite is concerned with who initiated the request. If it's a subresource, it looks at the embedding browsing contexts. But if it's a navigation, it could be the previous browsing context or even a different one altogether that triggered the navigation.

Reply:
Hmm, any suggestions? Most of the wording for same-site has been around browsing context, I didn't realize it was a term already in use.

Reply:
Yeah, I have been abusing this term for a while now... What we really mean is chain of all ancestor browsing contexts in the frame hierarchy (captured by Chromium's SiteForCookies), plus the request initiator. The "client" in Fetch contains approximately these things and is what RFC6265bis uses.

Maybe just "context"?

Reply:
I introduced a new term "same-site context" because I felt that "context" by itself was too overloaded. I'm not completely happy with it but it should get the point across.